libevmasm/Assembly.cpp

```diff
@@ -298,6 +423,43 @@ Json::Value Assembly::assemblyJSON(map<string, unsigned> const& _sourceIndices,
 	return root;
 }
 
+shared_ptr<Assembly> Assembly::loadFromAssemblyJSON(Json::Value const& _json, vector<string> const& _sourceList /* = {} */, bool _isCreation /* = true */)
```

Comment:
We already have these in the declaration. I think it's not a good idea to repeat them because they will easily go out of sync.
```suggestion
shared_ptr<Assembly> Assembly::loadFromAssemblyJSON(Json::Value const& _json, vector<string> const& _sourceList, bool _isCreation)
```